requirements-test.pip

```diff
@@ -1,3 +1,3 @@
 pytest>=2.9.0
-pytest-cov~=2.2.0
+pytest-cov
```

Comment:
Tady byl nějaký zádrhel s kontrolou pluginů pytestu - opravili to, ale v pozdější verzi pytest (a pytest-cov zřejmě blokoval upgrade).

Reply:
Please add exact version, this can cause problems in future.

Reply:
This is a test requirement - I don't think it will cause any trouble for production. And if it does cause trouble during tests, then at least we'll know something's wrong.

Reply:
Added an approximate version.